avm/res/network/application-gateway/main.bicep

```diff
@@ -368,21 +368,21 @@ resource applicationGateway_diagnosticSettings 'Microsoft.Insights/diagnosticSet
   }
 ]
 
-module applicationGateway_privateEndpoints '../../network/private-endpoint/main.bicep' = [
+module applicationGateway_privateEndpoints 'br/public:avm/res/network/private-endpoint:0.4.1' = [
   for (privateEndpoint, index) in (privateEndpoints ?? []): {
     name: '${uniqueString(deployment().name, location)}-applicationGateway-PrivateEndpoint-${index}'
     scope: resourceGroup(privateEndpoint.?resourceGroupName ?? '')
     params: {
-      // Variant 1: A default service can be assumed (i.e., for services that only have one private endpoint type)
-      name: privateEndpoint.?name ?? 'pep-${last(split(applicationGateway.id, '/'))}-${privateEndpoint.?service ?? '>defaultServiceName<'}-${index}'
+      // Variant 2: A default service cannot be assumed (i.e., for services that have more than one private endpoint type, like Storage Account)
```

Comment:
```suggestion
```

Reply:
Once we get the comments out the PR is good to go :)

Reply:
> Once we get the comments out the PR is good to go :)

@alexandersehr Comments removed!